src/flow_solver/flow_solver_cases/periodic_turbulence.cpp

```diff
@@ -734,53 +743,82 @@ void PeriodicTurbulence<dim, nstate>::compute_unsteady_data_and_write_to_table(
+        //if(do_calculate_numerical_entropy) this->add_value_to_data_table(current_numerical_entropy_change_FRcorrected,"numerical_entropy_scaled_this_tstep",unsteady_data_table);
+        if(is_rrk) this->add_value_to_data_table(relaxation_parameter, "relaxation_parameter",unsteady_data_table);
+        if (do_calculate_overintegrated_quantities) this->add_value_to_data_table(integrated_kinetic_energy,"kinetic_energy",unsteady_data_table);
+        if (do_calculate_overintegrated_quantities) this->add_value_to_data_table(integrated_enstrophy,"enstrophy",unsteady_data_table);
+        if(is_viscous_flow && do_calculate_overintegrated_quantities) this->add_value_to_data_table(vorticity_based_dissipation_rate,"eps_vorticity",unsteady_data_table);
+        if (do_calculate_overintegrated_quantities) this->add_value_to_data_table(pressure_dilatation_based_dissipation_rate,"eps_pressure",unsteady_data_table);
+        if(is_viscous_flow && do_calculate_overintegrated_quantities) this->add_value_to_data_table(strain_rate_tensor_based_dissipation_rate,"eps_strain",unsteady_data_table);
+        if(is_viscous_flow && do_calculate_overintegrated_quantities) this->add_value_to_data_table(deviatoric_strain_rate_tensor_based_dissipation_rate,"eps_dev_strain",unsteady_data_table);
         // Write to file
         std::ofstream unsteady_data_table_file(this->unsteady_data_table_filename_with_extension);
         unsteady_data_table->write_text(unsteady_data_table_file);
     }
     // Print to console
+    
     this->pcout << "    Iter: " << current_iteration
-                << "    Time: " << current_time
-                << "    Energy: " << integrated_kinetic_energy
-                << "    Enstrophy: " << integrated_enstrophy;
-    if(is_viscous_flow) {
+                << "    Time: " << current_time;
+    if (do_calculate_overintegrated_quantities) {
+        this->pcout << "    Energy: " << integrated_kinetic_energy
+                    << "    Enstrophy: " << integrated_enstrophy;
+    }
+    if(is_viscous_flow && do_calculate_overintegrated_quantities) {
         this->pcout << "    eps_vorticity: " << vorticity_based_dissipation_rate
                     << "    eps_p+eps_strain: " << (pressure_dilatation_based_dissipation_rate + strain_rate_tensor_based_dissipation_rate);
     }
     if(do_calculate_numerical_entropy){
-        this->pcout << "    Num. Entropy: " << std::setprecision(16) << numerical_entropy;
+        this->pcout << "    Num. Entropy cumulative, FR corrected: " << std::setprecision(16) << this->cumulative_numerical_entropy_change_FRcorrected; 
+        this->pcout << "    Num. Entropy change, FR corrected: " << std::setprecision(16) << current_numerical_entropy_change_FRcorrected; 
+        this->pcout << "    \\gamma \\Delta t sum bi v^T M u: " << std::setprecision(16) <<  dg->entropy_M_norm;
```

Comment:
rather than a latex math expression, can you put a descriptive name instead?

Reply:
Actually I will just remove this. I added it for a specific test and it's probably not useful enough to commit to master.